pricewars_merchant.py

```diff
@@ -125,8 +126,14 @@ def update_offers(self) -> None:
 
     def restock(self):
         order = self.producer.order(self.settings['restock limit'])
-        self.inventory_level += order.product.amount
```

Comment:
We should try to get rid of `amount` when we actually talk about the `number of` or `counts`.